tests/utils.go

```diff
@@ -695,7 +695,7 @@ func SynchronizedBeforeTestSetup() []byte {
 }
 
 func BeforeTestSuitSetup(_ []byte) {
-	rand.Seed(time.Now().Unix())
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I wonder if I should also multiply the result with the test-worker-index.

Reply:
yes

Reply:
do you think it will help?
Maybe we should make the names longer?

Reply:
Right now we only add 5 chars, maybe it's not enough?
`return name + "-" + rand.String(5)`

Reply:
Taking the node index directly now. That gives a unique value for each worker. I can create failing HostDisk tests easily with the same seed, but I can't reproduce the issue with the worker index or UnixNano granularity. I think it is at least good enough.

Reply:
Oh and btw, the duplicate VMI names is not an issue, the duplicate hostDisk directories for them are, and they use `rand.String(10)`.

Reply:
Should be ready now.